homeassistant/components/synology_dsm/__init__.py

```diff
@@ -479,16 +548,14 @@ def __init__(
         entity_info: Dict[str, str],
     ):
         """Initialize the Synology DSM entity."""
-        super().__init__()
-
         self._api = api
         self._api_key = entity_type.split(":")[0]
         self.entity_type = entity_type.split(":")[-1]
         self._name = f"{api.network.hostname} {entity_info[ENTITY_NAME]}"
-        self._class = entity_info[ENTITY_CLASS]
-        self._enable_default = entity_info[ENTITY_ENABLE]
-        self._icon = entity_info[ENTITY_ICON]
-        self._unit = entity_info[ENTITY_UNIT]
```

Comment:
Let's keep the `[]` since `.get()` frequently results in subtle bugs being introduced during refactoring.